Terminal.Gui/View/Layout/ViewLayout.cs

```diff
@@ -147,25 +146,22 @@ public virtual Rectangle FrameToScreen ()
     ///     View's <see cref="SuperView"/>'s <see cref="Viewport"/>.
     /// </summary>
     /// <returns>The coordinate relative to the <see cref="SuperView"/>'s <see cref="Viewport"/>.</returns>
-    /// <param name="x">Screen-relative column.</param>
-    /// <param name="y">Screen-relative row.</param>
-    public virtual Point ScreenToFrame (int x, int y)
+    /// <param name="location">Screen-relative coordinate.</param>
+    public virtual Point ScreenToFrame (in Point location)
     {
         if (SuperView is null)
         {
-            return new (x - Frame.X, y - Frame.Y);
+            return new (location.X - Frame.X, location.Y - Frame.Y);
```

Comment:
Offset here is also a good idea.

Reply:
![image](https://github.com/gui-cs/Terminal.Gui/assets/585482/961eb429-9e1e-4179-bc01-dd5f0f1b34e3)